tests/Caching/SQLiteJournal.phpt

```diff
@@ -21,7 +21,8 @@ class SQLiteJournalTest extends IJournalTestCase
 
 	public function createJournal()
 	{
-		return new SQLiteJournal;
+		static $id = 0;
+		return new SQLiteJournal(TEMP_DIR . '/sqlitejournal_' . ++$id . '.sqlite');
```

Comment:
It's not possible share :memory: SQLite nor after these changes:

https://github.com/nette/caching/pull/27/files#diff-88c1903cdb7494caec03523cf64621b1R121

Nor if I replace `BEGIN IMMEDIATE` with `BEGIN EXCLUSIVE`.